### DISCUSSION TITLE
Picture with multiple choice option

### DISCUSSION THREAD

**Discussion by obrien-avatech** [ditdot-dev/vue-flow-form#256] [Q&A] [answered]
Hello!

i am having an issue where my img is not displaying with QuestionType.MultiplePictureChoice.. i keep getting a broken link when using `imageSrc: '@/assets/img/promotional-eventtype-img.png'`. Anyone have an idea on how i can fix this?

**Comment by spinn** [comment]
Hey @obrien-avatech,

try it like this:

`imageSrc: './assets/img/promotional-eventtype-img.png'`

**Reply by obrien-avatech** [reply]
![image](https://user-images.githubusercontent.com/93557265/180492977-30e944b4-8fa5-4f01-8562-b8c838262ba2.png)
Getting a 404 with a broken link still
i tried changing the name of the .png incase it was stuck in cache somewhere.
tree structure 
-src
   -assets
       -img
           -event-promotional.png

**Comment by obrien-avatech** [comment, marked as answer]
please remove, no longer using this

**Comment by 6540140** [comment]
I use this type with absolute URL path in github.
like this..
####

          new QuestionModel({
            id: 'a030',
            type: QuestionType.MultiplePictureChoice,
            tagline: '2021학년도 서부초등드림스쿨 효과성 연구 설문', //질문 위에 표시할 문장
            title: "<article class='article-box'><div class='meta-cont'><span><a>밀란(Milan)과 마야(Maya)는 각자 자신이 원하는 모양의 카드를 만들기 위해 설문지에 답하였다. 밀란은 다음과 같이 선택하였다.<div id='container'><table class='content-table'><thead><tr><th> 1번 선택 : A</th><th> 2번 선택 : B</th><th> 3번 선택 : C</th><th> 4번 선택 : A</th></tr></thead></table></div><div align=center><img src='https://github.com/6540140/6540140/blob/main/dreamschool_samla2107_5-6_q1_1.gif?raw=true'></div>그리고 밀란은 자신의 선택에 따라 아래와 같은 카드를 받았다.</span><div align=center><img src='https://github.com/6540140/6540140/blob/main/dreamschool_samla2107_5-6_q1_1.png?raw=true'></div></div></article>",
            subtitle: "마야가 다음과 같이 선택하였을 때, 받게 될 카드는 무엇인가?<div><table class='content-table'><thead><tr><th> 1번 선택 : B</th><th> 2번 선택 : B</th><th> 3번 선택 : A</th><th> 4번 선택 : B</th></tr></thead></table></div>",
            placeholder: ' ',
            description: ' ',
            descriptionLink: ' ',
            helpText: '마야가 위와 같이 선택하였을 때, 받게 될 카드는 무엇인가?',
            helpTextShow: true,
            nextStepOnAnswer: false, 
            multiple: false,
            required: true,
            options: [
              new ChoiceOption({
                imageSrc: require('./assets/images/dreamschool_samla2107_5-6_q1_s1.png'),
                imageAlt: ' ',
                label: '1'
              }),
              new ChoiceOption({
                imageSrc: require('./assets/images/dreamschool_samla2107_5-6_q1_s2.png'),
                imageAlt: ' ',
                label: '2'
              }),
              new ChoiceOption({
                imageSrc: require('./assets/images/dreamschool_samla2107_5-6_q1_s3.png'),
                imageAlt: ' ',
                label: '3'
              }),
              new ChoiceOption({
                imageSrc: require('./assets/images/dreamschool_samla2107_5-6_q1_s4.png'),
                imageAlt: ' ',
                label: '4'
              }),
            ]
          }),
